From: Andrew Morton <akpm@linux-foundation.org>
Date: Sun, 4 May 2025 02:13:54 -0700
Subject: [PATCH] mm-add-zblock-allocator-fix-2

export try_alloc_pages_noprof() to modules for CONFIG_ZBLOCK=m

Cc: Vitaly Wool <vitaly.wool@konsulko.se>
Cc: Igor Belousov <igor.b@beldev.am>
Cc: Johannes Weiner <hannes@cmpxchg.org>
Cc: Nhat Pham <nphamcs@gmail.com>
Cc: Shakeel Butt <shakeel.butt@linux.dev>
Cc: Yosry Ahmed <yosry.ahmed@linux.dev>
Cc: David Hildenbrand <david@redhat.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 mm/page_alloc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mm/page_alloc.c b/mm/page_alloc.c
index a6fe1e9b9594..c77592b22256 100644
--- a/mm/page_alloc.c
+++ b/mm/page_alloc.c
@@ -7422,3 +7422,4 @@ struct page *try_alloc_pages_noprof(int nid, unsigned int order)
 	kmsan_alloc_page(page, order, alloc_gfp);
 	return page;
 }
+EXPORT_SYMBOL_GPL(try_alloc_pages_noprof);
-- 
2.39.5

